scripts/jsdoc.sh

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env bash
```

Comment:
Why is there a need for a bash script? I believe we can call `jsdoc -r -d doc/json public/externalLibs/*.js` from package.json itself.